owl2nl/src/main/java/org/aksw/owl2nl/OWLAxiomConverter.java

```diff
@@ -191,8 +189,33 @@ public void visit(OWLSubObjectPropertyOfAxiom axiom) {
 
 	@Override
 	public void visit(OWLEquivalentObjectPropertiesAxiom axiom) {
+		Set<OWLObjectPropertyExpression> ObjProperty = axiom.getProperties();
+		List<OWLObjectPropertyExpression> ObjProp = new LinkedList<>(ObjProperty);
```

Comment:
ArrayList is better for storing and accessing data while doubly LinkedList is better for manipulating data.